pkg/collect/collector_test.go

```diff
@@ -364,3 +365,83 @@ pwd=somethinggoeshere;`,
 		})
 	}
 }
+
+func TestCollector_DedupCollectors(t *testing.T) {
+	tests := []struct {
+		name       string
+		Collectors []*troubleshootv1beta2.Collect
+		want       []*troubleshootv1beta2.Collect
+	}{
+		{
```

Comment:
Maybe add a test case of our fictional collector that does not marshal to json?

Reply:
You have a recommendation on the "fictional collector" would look like since I'm using `troubleshootv1beta2.Collect` in the test?